Cargo.toml

```diff
@@ -1,49 +1,64 @@
-[package]
-name = "async-compression"
-version = "0.4.18"
+[workspace]
+members = ["crates/*"]
+resolver = "2"
+
+[workspace.package]
 authors = ["Wim Looman <wim@nemo157.com>", "Allen Bui <fairingrey@gmail.com>"]
 edition = "2018"
 license = "MIT OR Apache-2.0"
-keywords = ["compression", "gzip", "zstd", "brotli", "async"]
-categories = ["compression", "asynchronous"]
 repository = "https://github.com/Nullus157/async-compression"
+
+[workspace.dependencies]
+compression-codecs = { version = "0.1.0", path = "crates/compression-codecs" }
+compression-core = { version = "0.1.0", path = "crates/compression-core" }
+
+[package]
+name = "async-compression"
+version = "0.4.18"
 description = """
 Adaptors between compression crates and Rust's modern asynchronous IO types.
 """
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+keywords = ["compression", "gzip", "zstd", "brotli", "async"]
+categories = ["compression", "asynchronous"]
 
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
+default = ["all"]
```

Comment:
Hmmm why is we adding this default?

It would be too aggressive and opt in features users don't need

Reply:
Good catch. This was not intentional.